Bindicator.py

```diff
@@ -42,10 +68,19 @@ def startProgram():
     ##Connect to wifi and set Date Time
     #wifi.connect()
     wifi.setDateTime(10)
+
     ##Get Bin Data
     ##Check Against Date
     ##Light Sleep Untill Notification Dismissal or Date Change.
 
+def bin_color_display(key):
```

Comment:
Please keep Monash specific code in the Monash File. Different council areas have different Bin Color!